packages/core/data-transfer/lib/providers/local-file-destination-provider.ts

```diff
@@ -90,7 +97,18 @@ class LocalFileDestinationProvider implements IDestinationProvider {
     fs.mkdirSync(path.join(rootDir, 'configuration'));
   }
 
-  rollback(): void | Promise<void> {
+  close(): void {
+    const metadata = this.#providersMetadata.source;
+
+    if (metadata !== undefined) {
+      const metadataPath = path.join(this.options.file.path, 'metadata.json');
+      const data = JSON.stringify(metadata, null, 2);
+
+      fs.writeFileSync(metadataPath, data);
```

Comment:
This needs to make use of the writestream that all the other methods are using so that it can get handled by the same archive/compress/encrypt transforms

Reply:
You're totally right :sweat_smile:

Reply:
Done